src/main/java/se/diabol/jenkins/pipeline/DeliveryPipelineView.java

```diff
@@ -117,13 +112,20 @@
 
     private transient String error;
 
+    private static Random generator = new Random();
+    private String id = null;
+    private String viewUrl = null;
+    private boolean isPortletView = false;
```

Comment:
@tommysdk 
Let me check whether I can make another object for portlet configuration. Will let you know this weekend.